runtime/XsmmRunnerUtils.cpp

```diff
@@ -134,13 +134,11 @@ extern "C" int64_t _mlir_ciface_xsmm_matmul_dispatch(
   libxsmm_blasint k_int = k;
 
   libxsmm_gemm_shape l_shape;
-  libxsmm_bitfield l_flags;
-  if (isVNNI) {
+  libxsmm_bitfield l_flags = LIBXSMM_GEMM_FLAG_NONE;
+  if (flag == LIBXSMM_GEMM_FLAG_VNNI_B) {
```

Comment:
```suggestion
  if (flag && LIBXSMM_GEMM_FLAG_VNNI_B) {
```

Reply:
You probably want bitwise AND here:
` if (flag & LIBXSMM_GEMM_FLAG_VNNI_B) {`
If it still complains about implicit conversion, you might need to throw in `== 1` to check if it is set.

Reply:
I will try to OR prior to conversion to func. Will take a bit of time.

Reply:
yes, agree. There is not OR atm.

Reply:
> You probably want bitwise AND here

D'oh! That's what I meant. Sorry. Updated the comment.

Reply:
currently is a single flag

Reply:
I don;'t think the AND and the OR need to be done at the same time. You can easily check for the AND here and do the OR to set the flags later.